pkgs/by-name/sd/SDL2_classic/package.nix

```diff
@@ -86,19 +85,6 @@ stdenv.mkDerivation (finalAttrs: {
   ];
   outputBin = "dev"; # sdl-config
 
-  patches = [
-    # `sdl2-config --cflags` from Nixpkgs returns include path to just SDL2.
-    # On a normal distro this is enough for includes from all SDL2* packages to work,
-    # but on NixOS they're spread across different paths.
-    # This patch + the setup-hook will ensure that `sdl2-config --cflags` works correctly.
-    ./find-headers.patch
```

Comment:
you probably only intended to drop the commit that is now included in the release, not the header patch? Otherwise, please also remove the header patch file.

Reply:
Dropped by accident. Restored.